vendor/google.golang.org/grpc/balancer_conn_wrappers.go

```diff
@@ -25,83 +25,37 @@ import (
 	"google.golang.org/grpc/balancer"
 	"google.golang.org/grpc/connectivity"
 	"google.golang.org/grpc/grpclog"
+	"google.golang.org/grpc/internal/buffer"
```

Comment:
@YoyinZyc hi, I was looking at the couple of issue https://github.com/etcd-io/etcd/issues/11532 and https://github.com/etcd-io/etcd/issues/11529
and I think that these issues are might be because of https://github.com/etcd-io/etcd/tree/master/vendor/google.golang.org/grpc/internal doesn't have the referenced `internal/buffer` package. The https://github.com/grpc/grpc-go/tree/master/internal has it there. I haven't worked on updating etcd vendor files before but I believe having vendor dir updated with grpc-go master buffer should fix it. Please let me know what you think? Thanks!